### PR TITLE
Update neTactileActuators attribute on Escalators and Travelators #665tex_ifopt_equipmentAccess_version.xsd

### PR DESCRIPTION
**Pull request by ue71603**
fixes: https://github.com/NeTEx-CEN/NeTEx/issues/665

### CODE REVIEW

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd [hunk 1]
```diff
@@ -956,7 +956,7 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element name="TactileActuators" type="xsd:boolean" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Whether Lift has a tactile actuator.</xsd:documentation>
+					<xsd:documentation>Whether there are buttons to start the TRAVELATOR if it is not moving, and whether these buttons are intended and suited for the visually impaired.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd:959]: No, these here are the *TactileActuators* of the **LiftEquipment**. The change for Escalator and Travelator is already done (cf. line 726), I introduced it as a commit in #705.

**Review by trurlurl — CHANGES_REQUESTED** [review]

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd (hunk 1 above)
Comment (ue71603) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd:959]:
```suggestion
					<xsd:documentation>Whether there are buttons to start the LIFT if it is not moving, and whether these buttons are intended and suited for the visually impaired.</xsd:documentation>
```

Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd:959]: better with the new commit? @trurlurl

Comment (trurlurl) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd:959]: Given that in the *LiftButtonsGroup* we have *BrailleButtons*, *RaisedButtons*, and *TactileGroundFloorButton*, then the only meaning that seems to make sense for me would be something like: "Whether at least the call buttons have tactile markers for the visually impaired." OR **"Whether all buttons have tactile markers suited for the visually impaired."** The second one sounds better to me, but I don't know what the initial intention was.

**Review by trurlurl — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Pull request merged**